### PR TITLE
Code refactoring 

### PR DESCRIPTION
**Pull request by lvrfrc87**

### CODE REVIEW

File: ansible_collections/lvrfrc87/git_acp/plugins/module_utils/git_actions.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        command = ['git', 'add', '--']
+
+        command.extend(add)
+
+        rc, output, error = self.module.run_command(command, cwd=path)
+
+        if rc == 0:
+            return
+        else:
+            FailingMessage(self.module, rc, command, output, error)
```
Comment (chadell) [inline ansible_collections/lvrfrc87/git_acp/plugins/module_utils/git_actions.py:35]:
```suggestion
        FailingMessage(self.module, rc, command, output, error)
```

File: ansible_collections/lvrfrc87/git_acp/plugins/module_utils/git_configuration.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+class GitConfiguration:
+
+    def __init__(self, module):
+        self.module = module
+
+    def user_conifg(self):
```
Comment (chadell) [inline ansible_collections/lvrfrc87/git_acp/plugins/module_utils/git_configuration.py:10]: typo?

**Review by chadell — COMMENTED** [review]
code looks much cleaner now!

File: ansible_collections/lvrfrc87/git_acp/plugins/module_utils/git_configuration.py (hunk 2 above)
Comment (lvrfrc87) [reply ansible_collections/lvrfrc87/git_acp/plugins/module_utils/git_configuration.py:10]: typo...

**Pull request merged**